Common/cpp/Tools/RuntimeDecorator.cpp

```diff
@@ -72,6 +68,7 @@ void RuntimeDecorator::decorateRuntime(
 #endif // DEBUG
 
   jsi_utils::installJsiFunction(rt, "_log", logValue);
+  jsi_utils::installJsiFunction(rt, "_stringify", stringifyValue);
```

Comment:
As discussed offline, let's rename this function to `_toString` so it's consistent with JavaScript naming style.
```suggestion
  jsi_utils::installJsiFunction(rt, "_toString", stringifyValue);
```